js/buttons.js

```diff
@@ -0,0 +1,103 @@
+function addButtons() {
+  var buttonBox = waterUseViz.elements.buttonBox = svg.append('g')
+    .attr('id', 'button-container');
+  
+  buttonBox.append('rect')
+    .attr('id', 'button-background')
+    .attr('height', waterUseViz.dims.buttonBox.heightDesktop);
```

Comment:
should this be just the generic height as in `waterUseViz.dims.buttonBox.height`? That var adjusts based on the window size where `heightDesktop` is static, right?

Reply:
`waterUseViz.dims.buttonBox.height` isn't yet initialized at this point, which is why I had to use the default.